src/assets/scss/theme.scss

```diff
@@ -34,20 +40,25 @@ a:hover {
   border: 1px solid $dark;
 }
 .border-thick {
-  border-radius: 5px;
+  border-radius: 1px;
   border: 2px solid $dark;
 }
 
 .invert {
   background-color: $dark;
-  h1, h2, h3, h4, h5, p {
+  h1,
```

Comment:
Is this auto-format from new formatting config?

Reply:
from stylelint.
linting everything yesterday did not run stylelint :/
As I never really write .scss, I did not change in any way the default stylelint options.
https://stylelint.io/user-guide/configure